hr_expense_employee_analytic_default/README.rst

```diff
@@ -0,0 +1,107 @@
+=================================
+Expense Employee Analytic Default
+=================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:8a8ffb81895575fe038b644b6494e955d5d6c93d2137545974f178577af5fdac
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr--expense-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr-expense/tree/15.0/hr_expense_employee_analytic_default
+    :alt: OCA/hr-expense
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-expense-15-0/hr-expense-15-0-hr_expense_employee_analytic_default
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/hr-expense&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module allows to create Analytic Defaults Rules for employees to be applied to
+expenses.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to an employee (or create a new one) and set his private address.
```

Comment:
Let's use the company address instead of the private one for avoiding access errors.

Reply:
I'm sorry but I don't understand. The idea of this module is in fact to be able to set a default analytic account for each employee, if we don't use `address_home_id`, how do we do it?

Reply:
There are 2 linked partners on each `hr.employee`: the private one and the company one. Use the other. As reference, in v17 there's no more `address_home_id`.

Reply:
Ok, I guess you mean the `user_partner_id` field. It is already changed.